src-tauri/chain-spec.json

```diff
@@ -17,7 +17,7 @@
   "properties": {
     "ss58Format": 2254,
     "tokenDecimals": 18,
-    "tokenSymbol": "tSSC"
+    "tokenSymbol": "testnetSSC"
```

Comment:
Why this random change? It is called tSSC everywhere else (including Polkadot.js apps explorer and ss58-registry crate).

Reply:
This is what we discussed for the last 2 meetings. Jeremiah wanted us to be VERY explicit about these tokens have no value. 
But the symbol should have stayed same in this place, it's a victim of `replaceAll`. Reverting that specific instance